### PR TITLE
Derek Cruz | javascript-if.

### PR DESCRIPTION
**Pull request by Derek-Cruz**
JS If

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === string[0].toUpperCase()) {
```
Comment (BrettAlbright) [inline javascript-if/main.js]: so this is checking if the first letter is uppercase, instead, you want to check if the first letter is a capital `J`

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  }
+}
+
+function isOldEnoughToDrive(person) {
```
Comment (BrettAlbright) [inline javascript-if/main.js:34]: for this one, you want to check if they are 16 or over

File: javascript-if/main.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  if (person.age >= 21) {
+    return true;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 21) {
```
Comment (BrettAlbright) [inline javascript-if/main.js:41]: This function should return false no matter what argument is given

File: javascript-if/main.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    return !true;
+  }
+}
+
+function categorizeAcidity(pH) {
```
Comment (BrettAlbright) [inline javascript-if/main.js:46]: you have the first part, but you must also check for alkaline, acidic, and invalid values

File: javascript-if/main.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  if (pH === 7) {
+    return 'Neutral';
+  }
+}
+
+function introduceWarnerBro(name) {
+  if (name === 'Yakko') {
```
Comment (BrettAlbright) [inline javascript-if/main.js]: note that you want this returned if the brothers' names are used, but you need a different return if dot's name is given, and and a third one if none of the proper names are given

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a few updates good sir!

**Comment by Derek-Cruz** [comment]
JS If

File: javascript-if/main.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 21) {
+    return false;
+  }
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'Neutral';
+  } else if (pH >= 0 && pH <= 7) {
```
Comment (BrettAlbright) [inline javascript-if/main.js]: should be < 7 here as 7 is neutral

File: javascript-if/main.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    return 'acid';
+  } else if (pH >= 7 && pH <= 14) {
```
Comment (BrettAlbright) [inline javascript-if/main.js]: > 7 here, because again, 7 is neutral

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**